packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -265,19 +313,74 @@ class AndroidGradleBuilder implements AndroidBuilder {
       ...options, // suppresses gradle output.
       taskName,
     ];
+    preRunTask?.call();
 
-    RunResult result;
+    int exitCode = 1;
     try {
-      result = await _processUtils.run(
+      exitCode = await _processUtils.stream(
         command,
         workingDirectory: project.android.hostAppGradleRoot.path,
         allowReentrantFlutter: true,
         environment: _java?.environment,
+        mapFunction: consumeLog,
```

Comment:
Question (because I haven't used `ProcessUtils.stream`

The docs say
```
  /// If [filter] is null, no lines are removed.
  ///
  /// If [filter] is non-null, all lines that do not match it are removed. If
  /// [mapFunction] is present, all lines that match [filter] are also forwarded
  /// to [mapFunction] for further processing.
```

I'm assuming that if `filter` is null, all lines get passed to `mapFunction` (i.e., not just that no lines are removed, but that they are all forwarded?) Is that correct?

Reply:
(I noticed afterwards that this was just copied over, so if you happen to know feel free to reply but I'll assume yes because it was already written this way)